docs/what_is_data_science/Benefits of Data Science in the NHS.md

```diff
@@ -0,0 +1,9 @@
+# Benefits of Data Science in the NHS
+
+Data Science can be helpful in [solving a number of problems](./index.md). Specifically, this could lead to benefits like:
```

Comment:
im always in favor of saying "such as" instead of "like" in formal contexts but this is entirely personal preference

Reply:
haha I'm not a very formal person - but if you think it comes across better, happy to change it.